### PR TITLE
Switch to SWC

### PR DESCRIPTION
**Pull request by ferothefox**
This PR removes twin.macro and switches our React compiler to SWC. This cuts our incredibly fast build times in *half*. This is a draft until I can finish migrating the rest of the styles over though, the UI is odd and broken at place right now.

### CODE REVIEW

**Comment by ChecksumDev** [comment]
I find it really funny we originally were like "nah this isn't happening" and then you took some crazy pill and you did it anyways.

**Comment by ferothefox** [comment]
chardonnay and adderall

**Comment by ferothefox** [comment]
Ready for review 🚀

**Comment by ChecksumDev** [comment]
hUH

File: resources/scripts/components/dashboard/ServerRow.tsx [hunk 1]
```diff
@@ -174,7 +185,7 @@ export default ({ server, className }: { server: Server; className?: string }) =
                             ░░░░░░░▀▄▄░▒▒▒▒░░░░░░░░░░▒░░░█░
                             ░░░░░░░░░░▀▀▄▄░▒▒▒▒▒▒▒▒▒▒░░░░█░
                             ░░░░░░░░░░░░░░▀▄▄▄▄▄░░░░░░░░█░░ */}
```
Comment (ChecksumDev) [inline resources/scripts/components/dashboard/ServerRow.tsx:187]: What is this :trollface:

**Review by ChecksumDev — APPROVED** [review]
LGTM, but what is that ASCII art? :trollface:

File: resources/scripts/components/dashboard/ServerRow.tsx (hunk 1 above)
Comment (ferothefox) [reply resources/scripts/components/dashboard/ServerRow.tsx:187]: i dunno! :trollface:

**Pull request merged**